lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+    "name": "LinkedIn",
+    "origin": "https://www.linkedin.com",
+    "pageSize": {
+      "share": {
+        "height": 570,
+        "width": 520
+      }
+    },
+    "shareURL": "https://www.linkedin.com/shareArticle?mini=true&url=%{url}&title=%{title}&trk=FIREFOX_ACTIVATION",
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/linkedin.html"
+  },
+  "social.manifest.www-tumblr.com": {
+    "author": "Tumblr, Inc.",
+    "description": "Post text, photos, quotes, links, music, and videos from your browser, phone, desktop, email or wherever you happen to be. You can customize everything, from colors to your theme's HTML.",
+    "homepageURL": "https://www.tumblr.com",
+    "icon32URL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAOtJREFUeNpi/P//P8NAAiaGAQajDmAhpMDCPY5sw0/sXES5A/AAQST2e5qFAA6gDcRXkPiM9E4DFgOdCJMG0gEgy63olgvQQBoQTyaQIElKlKQ44CwQG+GQe4fGZ6RFFBiNzJIQCQjhCO5PQKxADwfgSlj/KSkJR2vDUQeQ44A/aHxuSqp1chzwBEtOikZzEE0dsAmL2AIgvg/ET4F4Ha0d0AXEb7GIgwojKSB2BGIRWjoA5EsvIH6OQ54ViO1p3SQ7BcRq0OrZHYjlgfgbtJm2Goi30NoBIPAFiPugmGzAONo1G3XAQDsAIMAAyz0m+77iTBQAAAAASUVORK5CYII=",
+    "icon64URL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAaJJREFUeNrsmzFLAzEUxxMp4lQEHR0ERRCKdFBwdBDq4CAoLpV2cPAjuBRxUcRR/AS6OwiCIq5SdFdc66R0KYKgIvEfmqGIYLTmztz7P/htL3fpj8u7d7mrNsYoydGjhAcFUAAFUAAFUAAFUAAFUIDQyP31AadLleCTrp8dRH0F3APzBSKWQD8YklwDxqUXwUnpAuYlCxgBs5IFbP7HviOpCS2BFamd4AI4lNgK58EeOAJ9YlphFztgzTU+vvFdN6hjErDOp0EKoAAKiCFCFUH9wwofrMrzCqAACqAACqAACqAACqCAdAU8eeQMZ1lAwyOnnGUB1x45G6q9c9w5p15QAoOxCzjxyLE/1u4eP4IrcOeWzqkKuKWelIBj0PTMHQBTYMxJsVGNXcAr2O5ifBFMxH4btO8ILroYX4ldwDtYBPVfjl/OQiPUAjNg1y0L37gBq1npBF9U+8XJKKiBS/D8KecN3IJ9MAcK4DzEZHIqvbAfS205tKv+eSfjQSX04VSaAjrDuNtkM+kTa/5pik+DFEABFEABFEABFEABMuNDgAEAKzQ9WAq/3UEAAAAASUVORK5CYII=",
+    "iconURL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAJNJREFUeNpi/P//PwMlgAWZY+EeR5SmEzsXwdlMeNQJAfFMKMYJ8BkQCcRpUEyyAc5A3ElyGECBCRDvBmJGJDFYSDMS4wI+bApJ8cI+LAYw4jKUiYFCQFMDvqKlCW5SDdiDxH4LxIdJNaAIiK8h8ZmBmIuYdAAD94BYB4hVgfg3EN8nNiExoCWgW/gUMFKanQECDABkohjLM1oQUAAAAABJRU5ErkJggg==",
+    "name": "Tumblr",
+    "origin": "https://www.tumblr.com",
+    "pageSize": {
+      "share": {
+        "height": 430,
+        "width": 550
+      }
+    },
+    "shareURL": "https://www.tumblr.com/share?v=3&u=%{url}&t=%{title}&s=%{description}",
+    "version": 1,
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/tumblr.html"
+  }
+};
+
+module.exports = DEFAULT_MANIFEST_PREFS;
```

Comment:
For anybody interested, here was my Joi schema validator script thing:

```js
const Joi = require('joi');

const manifests = require('./ShareManifests');

function lintManifest(manifest, cb) {
  const dataUriSchema = Joi.string().uri({
    scheme: ['data']
  });
  const itemSchema = Joi.object().keys({
    author: Joi.string().required(),
    description: Joi.string().required(),
    homepageURL: Joi.string().uri().optional(), // OPTIONAL
    iconURL: dataUriSchema.required(),   // NOTE: `data:` URI
    icon32URL: dataUriSchema.required(), // NOTE: `data:` URI
    icon64URL: dataUriSchema.required(), // NOTE: `data:` URI
    name: Joi.string().required(),
    origin: Joi.string().uri().required(),
    pageSize: Joi.object().optional().keys({ // OPTIONAL
      share: Joi.object().required().keys({
        height: Joi.number().integer().required(),
        width: Joi.number().integer().required()
      })
    }),
    postActivationURL: Joi.string().uri().required(),
    shareURL: Joi.string().required(), // This isn't a `uri()` because of odd tokens.
    version: Joi.number().optional() // OPTIONAL
  });

  // HACK: I'm a terrible QA and person.
  const obj = {};
  Object.keys(manifest).forEach((site) => obj[site] = itemSchema);

  Joi.validate(manifest, Joi.object().keys(obj), {abortEarly: false}, cb);
}

lintManifest(manifests, (err, value) => {
  if (err) {
    const messages = err.details.map(({message, path}) => `${path}: ${message}`);
    console.error(messages.join('\n'));
    process.exit(1);
  }
  console.log(`manifest is valid!`);
});
```

### Output:
```
social.manifest.twitter-com.icon32URL: "icon32URL" is required
social.manifest.twitter-com.icon32ULR: "icon32ULR" is not allowed
social.manifest.mg-mail.yahoo.com.iconURL: "iconURL" must be a valid uri with a scheme matching the data pattern
social.manifest.mg-mail.yahoo.com.icon32URL: "icon32URL" must be a valid uri with a scheme matching the data pattern
social.manifest.mg-mail.yahoo.com.icon64URL: "icon64URL" must be a valid uri with a scheme matching the data pattern
```